cmd/mavenExecuteIntegration_test.go

```diff
@@ -12,6 +12,7 @@ type mavenExecuteIntegrationTestUtilsBundle struct {
 }
 
 func TestIntegrationTestModuleDoesNotExist(t *testing.T) {
+	t.Parallel()
```

Comment:
Not sure if this has an effect in test functions that don't execute multiple runs.

Reply:
Not sure either. Can multiple test functions run in parallel?

Reply:
When I read https://eleni.blog/2019/05/11/parallel-test-execution-in-go/, my whole expectations and also implementation was wrong. Let me correct it for this PR.

Reply:
In the official documenation it makes clear how t.parallel is used in t.run (https://golang.org/pkg/testing/#T.Run)
In stackoverflow it is also shown that this also work across test methods (https://stackoverflow.com/questions/44325232/are-tests-executed-in-parallel-in-go-or-one-by-one)

Reply:
Thanks! Great, I learned two things via this PR! I've been doing it wrongly for all my tests so far. Not the test data detaching, but not calling Parallel() in each run and also not calling Parallel for functions that have no runs.